src/core/EdFi.DataManagementService.Core/Response/FailureResponse.cs

```diff
@@ -123,6 +126,29 @@ public static FailureResponse ForDataConflict(string dependentItemName)
         );
     }
 
+    public static FailureResponse ForInvalidReferences(ResourceName[] resourceNames) =>
+        new(
+            detail:
+            $"The referenced {string.Join(",", resourceNames.Select(x => x.Value))} item(s) do not exist.",
+            type: $"{BaseTypePrefix}:data-conflict:unresolved-reference",
+            title: "Unresolved Reference",
+            status: 409,
+            correlationId: null,
+            validationErrors: null,
+            errors: null
+        );
+
+    public static FailureResponse ForKeyChangeNotSupported(string error) =>
```

Comment:
I'd prefer for this method name to use the DMS vocabulary e.g. `ForImmutableIdentity()`

Reply:
You didn't change this, and as a result the code is confusing and wrong.